src/de/jost_net/JVerein/io/ZipMailer.java

```diff
@@ -66,6 +65,7 @@ public ZipMailer(final File zipfile, final String betreff, final String text, fi
       @Override
       public void run(ProgressMonitor monitor) throws ApplicationException
       {
+        ZipFile zip = null;
         try
```

Comment:
ZipFile ist [CloseAble](https://docs.oracle.com/javase/8/docs/api/java/io/Closeable.html). Da kann man seit Java 8 [try with resources](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html) machen.
```suggestion
        try(ZipFile zip = new ZipFile(zipfile))
```

Reply:
Danke für den tip, das kannte ich noch nicht. Habe ich geändert